models/game.py

```diff
@@ -0,0 +1,40 @@
+# Initialising Game model
+
+class Game(object):
+	def __init__(self):
+		self.id = None
+		self.slot = None
+		self.try_at = None
+		self.chance = None
+		self.playerId = None
+
+	def setId(self,id):
```

Comment:
It's always best to never use setters unless it is required. That way the object becomes kinda immutable (at least in Java)